org.tryton.Tryton-66.yml

```diff
@@ -0,0 +1,31 @@
+id: org.tryton.Tryton-66
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+command: tryton
+build-options:
+  share: network
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland?

Reply:
It uses GTK and we have user using it with wayland.

Reply:
```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

then this.